modepy/matrices.py

```diff
@@ -367,12 +482,7 @@ def modal_quad_mass_matrix(
 
     .. versionadded :: 2024.2
     """
-    modal_mass_matrix = np.empty((len(test_functions), len(quadrature.weights)))
-
-    for i, test_f in enumerate(test_functions):
-        modal_mass_matrix[i] = test_f(quadrature.nodes) * quadrature.weights
-
-    return modal_mass_matrix
+    return modal_quadrature_operator(quadrature, test_functions)
```

Comment:
If this `modal_quadrature_operator` is the same as the `modal_quad_mass_matrix`, why introduce a new function? (Same for `nodal_quad_mass_matrix`)

Reply:
Mostly because of the name. As a user, `*_quad_mass_matrix` would make me assume this is only used for generating an operator whose action is equivalent to applying a mass matrix. 

I suppose I can set `*_quad_mass_matrix = *_quadrature_operator` and get rid of `*_quad_mass_matrix` functions?

Reply:
I don't know, I'm personally not very bothered by the existing naming, especially since it gives a pretty clear idea of some use for the function (i.e. everybody knows what a mass matrix is), while "quadrature_operator" is very generic sounding.

> I suppose I can set *_quad_mass_matrix = *_quadrature_operator and get rid of *_quad_mass_matrix functions?

If these go away, the nice way to do it is to add a `warn("This function is going away on {deadline}. Use {other_function} instead.")` sort of thing so that users have time to migrate. :grin:

Reply:
Sure, I agree. I guess my main point is that `*_quadrature_operator` can be used to generate a stiffness matrix and a mass matrix while `*_quad_mass_matrix` is restricted to only generating a mass matrix (since it requires the vandermonde built with `test_functions` is invertible).

I agree that the naming is pretty generic. I'm open to suggestions if you've got any! :)

Reply:
Also, if you do decide to deprecate the `*_quad_mass_matrix`, I would suggest using `*_quad_operator` for the new names, since most of the functions in here seem to use the short name?